src/com/lftechnology/java/training/krishna/basics/evenfibonacci/EvenFibonacci.java

```diff
@@ -0,0 +1,60 @@
+package com.lftechnology.java.training.krishna.basics.evenfibonacci;
+
+/**
+ * EvenFibonacci.java
+ * 
+ * The EvenFibonacci program implements an application that simply sum of the even-valued terms
+ * 
+ * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+ * @version 1.0
+ */
+public class EvenFibonacci {
+	private int firstNumber;
+	private int secondNumber;
```

Comment:
Do we really need to make first number and second number a class variable. For fibonacci series the first two numbers are fixed. unless the question also allow to sum fibonacci numbers between a specified range